src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1123,10 +1207,12 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
 
     ucs_debug("%s: opened DEVX md", ibv_get_device_name(ibv_device));
 
-    dev->flags     |= UCT_IB_DEVICE_FLAG_MLX5_PRM;
-    md->flags      |= UCT_IB_MLX5_MD_FLAG_DEVX;
-    md->flags      |= UCT_IB_MLX5_MD_FLAGS_DEVX_OBJS(md_config->devx_objs);
-    md->super.name  = UCT_IB_MD_NAME(mlx5);
+    dev->flags          |= UCT_IB_DEVICE_FLAG_MLX5_PRM;
+    md->flags           |= UCT_IB_MLX5_MD_FLAG_DEVX;
+    md->flags           |= UCT_IB_MLX5_MD_FLAGS_DEVX_OBJS(md_config->devx_objs);
+    md->super.name       = UCT_IB_MD_NAME(mlx5);
+    md->super.cap_flags |= cap_flags;
+    md->super.vhca_id    = vhca_id;
```

Comment:
can we remove the local vars cap_flags, vhca_id and save the values directly to md->super?